jdk17.sh

```diff
@@ -2,8 +2,19 @@
 set -euxo pipefail
 
 apt-get update
+apt-get -y dist-upgrade
 DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends \
-  openjdk-17-jdk-headless \
-  gradle
+  curl openjdk-17-jdk-headless unzip
 apt-get clean
 rm -rf /var/lib/apt/lists/*
+
+curl -sLo/tmp/gradle.zip https://services.gradle.org/distributions/gradle-7.5-rc-2-bin.zip
```

Comment:
Do you mind switching it to gradle-7.4.2? Gradle 7.5 is still a release candidate at the moment.

Reply:
sure, done